.github/workflows/ci.yaml

```diff
@@ -17,6 +17,11 @@ jobs:
       - run: go mod download
       - run: make fmt
       - run: make tidy
+      - run: |
```

Comment:
If/when we move to hosted `.zip` binaries like we do for Zeek, we can remove this and install into `zq/bin` via the Makefile (like for Zeek).